public/app/plugins/datasource/prometheus/partials/config.html

```diff
@@ -1,3 +1,14 @@
 <datasource-http-settings current="ctrl.current" suggest-url="http://localhost:9090">
 </datasource-http-settings>
 
+<h3 class="page-heading">Prometheus details</h3>
+
+<div class="gf-form-group max-width-30">
+	<div class="gf-form">
+		<label class="gf-form-label width-13">Label Keys</label>
+		<input type="text" class="gf-form-input max-width-18" ng-model='ctrl.current.jsonData.labelKeys' placeholder="job,instance"></input>
```

Comment:
I reconsider about the possibility to get all label keys from query response.
And it is NO. Some of the label keys can get from response.
But, if the query use "count" or aggregate function, the labels which metrics has, is not appeared in query response...

Gathering label keys from response could improve coverage of label keys.
But, need to input box too. I'll add it.